compiler/src/dotty/tools/backend/jvm/BCodeHelpers.scala

```diff
@@ -406,8 +612,9 @@ trait BCodeHelpers extends BCodeIdiomatic with BytecodeWriters {
      * must-single-thread
      */
     def getExceptions(excs: List[Annotation]): List[String] = {
-      for (ThrownException(exc) <- excs.distinct)
-      yield internalName(exc)
+      // for (ThrownException(exc) <- excs.distinct)
+      // yield internalName(exc)
```

Comment:
Why is this commented out?

Reply:
Inlined `ThrownException` which always returned `None`. Kept the code for reference. Added a TODO.